src/java/org/apache/cassandra/cql3/statements/schema/CreateTableStatement.java

```diff
@@ -372,16 +372,26 @@ else if (!builder.hasRegularColumns())
     @Override
     public Set<String> clientWarnings(KeyspacesDiff diff)
     {
+        Set<String> warnings = new HashSet<>();
+
         int tableCount = Schema.instance.getNumberOfTables();
         if (tableCount > DatabaseDescriptor.tableCountWarnThreshold())
         {
             String msg = String.format("Cluster already contains %d tables in %d keyspaces. Having a large number of tables will significantly slow down schema dependent cluster operations.",
                                        tableCount,
                                        Schema.instance.getKeyspaces().size());
             logger.warn(msg);
-            return ImmutableSet.of(msg);
+            warnings.add(msg);
         }
-        return ImmutableSet.of();
+
+        if (attrs.hasUnsupportedDseCompaction())
+        {
+            warnings.add("The given compactin strategy is not supported. The compaction strategy parameter was " +
```

Comment:
typo

Reply:
could we log also the name of this unsupported compacting strategy?

Reply:
Unfortunately not without changing the API, so I worked around it with "The given compaction strategy". 
Fixed the typo and added the strategy to the message.